deployment/aws/lambda/Dockerfile

```diff
@@ -1,15 +1,7 @@
-FROM lambci/lambda:build-python3.8
+FROM public.ecr.aws/lambda/python:3.8
 
-WORKDIR /tmp
+RUN pip install titiler.application mangum>=0.10.0
 
-RUN pip install titiler.application mangum>=0.10.0 -t /var/task --no-binary numpy,pydantic
+COPY lambda/handler.py ${LAMBDA_TASK_ROOT}/
 
-# Reduce package size and remove useless files
-RUN cd /var/task && find . -type f -name '*.pyc' | while read f; do n=$(echo $f | sed 's/__pycache__\///' | sed 's/.cpython-[2-3][0-9]//'); cp $f $n; done;
-RUN cd /var/task && find . -type d -a -name '__pycache__' -print0 | xargs -0 rm -rf
-RUN cd /var/task && find . -type f -a -name '*.py' -print0 | xargs -0 rm -f
-RUN find /var/task -type d -a -name 'tests' -print0 | xargs -0 rm -rf
```

Comment:
This could still be kept, In addition of removing useless files it make startup faster!

Reply:
To reduce the docker image size, we'd have to do it in the same RUN as the pip install, but I'd want to see how much space/time it saves for this.